packages/nocodb-sdk/src/lib/formulaHelpers.ts

```diff
@@ -60,7 +67,11 @@ export async function substituteColumnAliasWithIdInFormula(
   };
   // register jsep curly hook
   jsep.plugins.register(jsepCurlyHook);
-  const parsedFormula = jsep(formula);
+  const parsedFormula = jsep(
+    // formula may include double curly brackets in previous version
+    // convert to single curly bracket here for compatibility
+    formula.replaceAll('{{', '{').replaceAll('}}', '}')
+  );
```

Comment:
What if `{{` or `}}` is part of a string ?

Reply:
I believe now this replace is no longer required as it has been catered inside the curly hook.